exercises/acronym/example.rs

```diff
@@ -0,0 +1,27 @@
+///
+/// Techies love their TLA (Three Letter Acronyms)!
+///
+/// Help generate some jargon by writing a program that converts a long name
+/// like Portable Network Graphics to its acronym (PNG).
+#[allow(unused_variables)]
+pub fn abbreviate(phrase: &str) -> String {
+    phrase.split(|c: char| c.is_whitespace() || !c.is_alphanumeric())
+          .flat_map(|word| split_camel(word))
+          .filter(|word| !word.is_empty())
+          .map(|word| word.split_at(1).0.to_uppercase())
+          .collect::<String>()
```

Comment:
I think the `::<String>`` is not necessary since the compiler knows by using the fact tha `abbreviate` is `-> String`.

That said I dont mind if it stays.